recipes/bvpy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "bvpy" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.inria.fr/mosaic/bvpy/-/archive/master/bvpy-master.tar.gz
```

Comment:
We don't package `master` as `1.0.0`

Reply:
Ok, can I use instead the commit associated to the tag `1.0.0.` (https://gitlab.inria.fr/mosaic/bvpy/-/archive/1.0.0/bvpy-1.0.0.tar.gz) ?
Thank you for your time.